hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/SCMClientProtocolServer.java

```diff
@@ -587,6 +588,16 @@ public void deleteContainer(long containerID) throws IOException {
     }
   }
 
+  @Override
+  public Map<String, List<ContainerID>> getContainersOnDecomNode(DatanodeDetails dn) throws IOException {
+    try {
+      return scm.getScmDecommissionManager().getMonitor().getContainersReplicatedOnNode(
```

Comment:
I think it would be cleaner if we did not expose the monitor outside of ScmDecommissionManager. Instead, create a method on ScmDecommissionManager like `getContainersReplicatedOnNode(DatanodeDetails dn)` and have it return the details it gets from the monitor.

Reply:
Thank you for the review! I have made the change in my recent commit to not expose the monitor.